src/Bicep.Core/FileSystem/IFileResolver.cs

```diff
@@ -46,11 +46,16 @@ public interface IFileResolver
         IEnumerable<Uri> GetFiles(Uri fileUri, string pattern);
 
         /// <summary>
-        /// Check whether specified URI exsists (depends on URI types). fileUri MUST have a trailing '/'
+        /// Check whether specified URI's directory exists if specified URI is a file:// URI. fileUri MUST have a trailing '/'
         /// </summary>
         /// <param name="fileUri">The fileUri to test</param>
-        bool TryDirExists(Uri fileUri);
+        bool DirExists(Uri fileUri);
 
+        /// <summary>
+        /// Checks if the specified URI exists.
```

Comment:
Specify here that this only works for files, not dirs?

Reply:
Done